akka-stream/src/main/scala/akka/stream/javadsl/Source.scala

```diff
@@ -281,6 +281,24 @@ object Source {
   }
 
   /**
+   * Combine the elements of multiple streams into a stream of lists.
+   */
+  def zipN[T](sources: java.util.List[Source[T, _ <: Any]]): Source[java.util.List[T], NotUsed] = {
+    import scala.collection.JavaConverters._
+    val seq = if (sources != null) sources.asScala.map(_.asScala) else Seq()
```

Comment:
I have to admit that my implementation of `zipN` is deeply inspired from the `combine` one, so I use the standard library to convert collections from java to scala and I support `null`. I wrote it this way in order to be consistent with the existing code, should I update it?